multiversx_sdk_network_providers/api_network_provider.py

```diff
@@ -118,14 +122,47 @@ def get_transaction(self, tx_hash: str) -> TransactionOnNetwork:
 
         return transaction
 
-    def get_transactions(self, address: IAddress, pagination: IPagination = DefaultPagination()) -> List[TransactionOnNetwork]:
+    def get_account_transactions(self, address: IAddress, pagination: IPagination = DefaultPagination()) -> List[TransactionOnNetwork]:
```

Comment:
Breaking change, we should bump at least the minor version e.g. `v0.8.0` (since we are still at `v0.*`) and document this - especially because the name `get_transactions` now means something else.

Reply:
Bumped the version and also documented a bit in the PR description.